_lessons/dbt-project-design.html

```diff
@@ -641,11 +889,12 @@ <h3 style="text-align: center;"><span style="color: green;">source</span>
 ]
 
 ???
-dbt models are like kitchen tools. The only single-purpose kitchen appliance
-you should ever have is a rice cooker.
+[Teacher Note:]  
+You can think of dbt as our assembly line which produces expected outputs every  
+time.  
 
-BI assets are like recipes. It doesn't hurt as much to have similar ones, but
-when it comes time to make dinner, your guests might become confused :)
+You can think of the BI layer as the place where we take the items produced from  
+our assembly line to customize them in order to meet our stakeholder's needs. 
```

Comment:
praise: this is an awesome analogy! that really hits home.

in the future, I wonder if we can supplement that with a visual of sorts.